File: FE/app/types/models/LoginModel.ts
// ========== USER ==========
import type {LoginTokenModel} from "~/types/models/TokenModel";

export interface LoginModel {
    userName?: string | null;
    email?: string | null;
    role?: string[] | null;
    status?: string | null;
    isVerified?: boolean | null;
    token?: LoginTokenModel | null

}

File: BE/src/main/kotlin/ngga/ring/myblog/data/auth/models/LoginModel.kt
package ngga.ring.myblog.data.auth.models

data class LoginModel(
    var userName: String? = null,
    var email: String? = null,
    var role: List<String>? = null,
    var status: String? = null,
    var isVerified: Boolean? = false,
    val token: TokenModel? = null
)